### PR TITLE
add regression fine-tuning

### PR DESCRIPTION
**Pull request by fancyerii**
I want to use bert to calculate the similarity of two sentences and output a similarity score between 0 and 1. 

So I change the output from num_classes to 1 and use sigmoid instead of softmax. The loss function change from cross entropy to mse. 

Also the input is not a label(int32) but a float32 value.

### CODE REVIEW

**Comment by fancyerii** [comment]
I signed it!

**Comment by nomemm** [comment]
> I want to use bert to calculate the similarity of two sentences and output a similarity score between 0 and 1.
> 
> So I change the output from num_classes to 1 and use sigmoid instead of softmax. The loss function change from cross entropy to mse.
> 
> Also the input is not a label(int32) but a float32 value.

Hi,

Thank you!

It seems, the code under "   if FLAGS.do_predict:" in the run_reg.py file should be modified as well. 
now it outputs -1 instead of similarity value.
